### PR TITLE
Portfolio: Finish mobile version

### PR DESCRIPTION
**Pull request by maximoortelli**
**### Release Implementations:**

- In this new Pull Request I open the entire mobile portfolio, updating the html code, css and adding innovative images.

- README file updated successfully.

- Added "project" and "About me" sections.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -53,12 +53,305 @@ <h3>LET'S CONNECT</h3>
...
+
+             <li class="abilities flex-row">
+                 <p id="frameworks">Frameworks</p><img class="next2" src="images/next.png" alt="clikleable arrow">
+             </li>
+
+             <li class="abilities flex-row">
+                 <p id="skills">Skills</p><img class="next2" src="images/next.png" alt="clikleable arrow">
+             </li>
+
+         </ul>
```
Comment (MaganaAsiati) [inline index.html:353]: - Kindly make sure you have those images in your images folder, If you check very well in your images folder there is a non-image called, next.png or next down.png. kindly fix that to make your pages look nice by removing those broken links for images. :+1: 

image

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 

Hi @maximoortelli 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlonzoLeroneThisCloseGIF

## Highlights
✅ No linter error
✅ README is professional

## Required Changes ♻️

- Great job so far, your code looks very nice, however, there is the useless repetition of `<p>`, kindly make sure you use `<p> ` tag for a paragraph. please follow [this links](https://www.semrush.com/blog/html-tags-list/?kw=&cmp=Africa_SRCH_DSA_Blog_EN&label=dsa_pagefeed&Network=g&Device=c&utm_content=654619918451&kwid=dsa-1753200731733&cmpid=18364843126&agpid=141057035883&BU=Core&extid=60114153695&adpos=&gad=1&gclid=Cj0KCQjwr82iBhCuARIsAO0EAZwFCK79pn7gbWVVvcAxiflrvFCNMNC-DK0GiOINwd4JSoPkuXLpRBQaAk6NEALw_wcB) to learn more about how to use different tags in html:+1: 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 1 above)
Comment (maximoortelli) [reply index.html:353]: Thank you very much for your contribution to the project, it has been corrected satisfactorily, I send you a cordial greeting.

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @maximoortelli , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Well done working on the requested change. ✔
- Your project now looks very nice :+1:  . ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**